docs/reference/known_limitations.md

```diff
@@ -15,25 +15,35 @@ We are developing a new feature that will allow you to modify these settings dir
 
 ## Backups
 
-The limitations related to backups in Percona Everest are as follows:
+Let's delve into the limitations of on-demand backups in Percona Everest. 
+
+### PostgreSQL limitations for on-demand backups
 
 - When attempting to delete a PostgreSQL database that contains backups created with Everest versions older than 1.0.0, the database may become stuck in the **Deleting** state. 
 
     **Workaround**: To prevent this, manually delete any backups created with versions prior to 1.0.0 by using the **Delete** action on the **Backups** page before deleting the database.
 
-- You cannot change the bucket name for a specific backup storage. Doing so will make any backups taken for that bucket unusable.
+- You cannot change the bucket name and region for a specific backup storage. Doing so will make any backups taken for that bucket unusable.
```

Comment:
@oksana-grishchenko Haven't we enforced this in the UI/API already?

Reply:
@recharte from the UI side, we did.

Reply:
In this case I don't think we should mention the last sentence. If we don't allow it to be change the consequences never happen.
```suggestion
- You cannot change the bucket name and region for a specific backup storage.
```

Reply:
Actually I'm not sure anymore :sweat_smile: , I leave it up to Rasika... Keeping the info of what would happen if we did allow it is also fine. You can ignore this comment.